docs/ops/broadcast_rules.md

```diff
@@ -0,0 +1,9 @@
+# Broadcast rules for elementwise ops {#openvino_docs_ops_broadcast_rules}
```

Comment:
`Broadcast rules for elementwise ops` -> `Broadcast Rules For Elementwise Operations`

Reply:
Corrected.